phoenix-core/src/it/java/org/apache/phoenix/rpc/UpdateCacheIT.java

```diff
@@ -87,7 +87,7 @@ public void testUpdateCacheForTxnTable() throws Exception {
             String fullTableName = INDEX_DATA_SCHEMA + QueryConstants.NAME_SEPARATOR + tableName;
             Connection conn = DriverManager.getConnection(getUrl(), PropertiesUtil.deepCopy(TEST_PROPERTIES));
             conn.createStatement().execute("create table " + fullTableName + TestUtil.TEST_TABLE_SCHEMA + "TRANSACTIONAL=true,TRANSACTION_PROVIDER='" + provider + "'");
-            helpTestUpdateCache(fullTableName, new int[] {1, 1}, false);
+            helpTestUpdateCache(fullTableName, new int[] {1, 3}, false);
```

Comment:
What is the reason for this increase from 1 to 3 in expected RPCs?

Reply:
I see that the same test for non TX table also needs 3 RPCs. I assume our change eliminated this discrepancy.